iree_tests/benchmarks/benchmark_sdxl_cpu.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# Copyright 2024 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+set -xeuo pipefail
+
+this_dir="$(cd $(dirname $0) && pwd)"
```

Comment:
nit/tip: it's common in some scripts to differentiate between local variables and constants / environment variables. Here you could use ALL_CAPS for these variables defined at the top that are constant throughout the rest of the script

`THIS_DIR`, `IREE_ROOT`, `VAE_DECODE_DIR`, etc.

https://google.github.io/styleguide/shellguide.html#constants-and-environment-variable-names